oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/TraverseAndSortTask.java

```diff
@@ -176,7 +181,17 @@ public List<File> call() {
 
             return sortedFiles;
         } catch (IOException e) {
-            log.error(taskID + " could not complete download ", e);
+            log.error(taskID + " could not complete download with ", e);
+        } catch (Exception e) {
```

Comment:
This will not be `EOFException` because we do want to skip those, and will catch the __merge failure__ case which originally was throwing just `Exception` and now I change it to `RuntimeException` (correct me if there's a better one... not sure what is correct to use here). So changing it to now catch `RuntimeException` instead.